src/components/formatters/FormattedNumber.js

```diff
@@ -1,12 +1,9 @@
-import React from 'react'
+import i18n from '@dhis2/d2-i18n'
```

Comment:
Shouldn't this one come from `/src/locales/index.js`?
cc @amcgee I'm really not sure about this one, but I remember that this was an issue once

Reply:
Applies to all files importing from this module

Reply:
In apps we only need one import to `src/locales` (which I do [here](https://github.com/dhis2/data-administration-app/blob/de0e9f560e81f4457f7be5d87d2836d8a2523dd6/src/App.js)), it's in libraries (which are tree shaken) that every `i18n` import should be from that file.